espnet2/utils/eer.py

```diff
@@ -0,0 +1,96 @@
+"""
```

Comment:
ditto
add a test.

It is especially important since the objective scoring MUST NOT have any errors.